src/components/history-page/history-page.tsx

```diff
@@ -195,35 +33,31 @@ export const HistoryPage: React.FC = () => {
                      })
   }, [allEntries])
 
-  const entriesToShow = useMemo<LocatedHistoryEntry[]>(() =>
+  const entriesToShow = useMemo<HistoryEntry[]>(() =>
       sortAndFilterEntries(allEntries, toolbarState),
     [allEntries, toolbarState])
 
-  return <Fragment>
-    <ErrorModal show={ error !== '' } onHide={ resetError }
-                titleI18nKey={ error !== '' ? `landing.history.error.${ error }.title` : '' }>
-      <h5>
-        <Trans i18nKey={ error !== '' ? `landing.history.error.${ error }.text` : '' }/>
-      </h5>
-    </ErrorModal>
-    <h1 className="mb-4"><Trans i18nKey="landing.navigation.history"/></h1>
-    <Row className={ 'justify-content-center mt-5 mb-3' }>
-      <HistoryToolbar
-        onSettingsChange={ setToolbarState }
-        tags={ tags }
-        onClearHistory={ clearHistory }
-        onRefreshHistory={ refreshHistory }
-        onExportHistory={ exportHistory }
-        onImportHistory={ importHistory }
-        onUploadAll={ uploadAll }
+  useEffect(() => {
+    refreshHistoryState().catch(
+      showErrorNotification(t('landing.history.error.getHistory.text'))
+    )
+  }, [t])
+
+  return (
+    <Fragment>
+      <h1 className="mb-4">
+        <Trans i18nKey="landing.navigation.history"/>
+      </h1>
+      <Row className={ 'justify-content-center mt-5 mb-3' }>
+        <HistoryToolbar
+          onSettingsChange={ setToolbarState }
+          tags={ tags }
```

Comment:
`tags` is used only in the toolbar. How about moving the tags var into the toolbar component?

Reply:
Moved.